stingray/pulse/pulsar.py

```diff
@@ -617,30 +632,25 @@ def get_orbital_correction_from_ephemeris_file(mjdstart, mjdstop, parfile,
         Function that accepts times in MJDs and returns the deorbited times.
     """
     from scipy.interpolate import interp1d
+    import astropy.units as u
```

Comment:
Importing a library as a single character is a bit risky, and PEP8 suggests using single-characters only for indices, I think. Might be worth importing as `units`, rather than just `u`.